backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/api/impl/DeliveryDayGenerator.java

```diff
@@ -151,6 +150,26 @@ public final void generate(final String trxName)
 		}
 	}
 
+	@Override
+	public void generateDeliveryDaysForTourVersion(final String trxName, final I_M_TourVersion tourVersion)
```

Comment:
see if you can't avoid the code duplication; looks very similar to `generateForTour` to me on the first glimpse

Reply:
Extracted duplicate code to new method!